src/legacy/core_plugins/metrics/public/components/aggs/top_hit.js

```diff
@@ -34,60 +35,111 @@ import {
   EuiFormRow,
 } from '@elastic/eui';
 import { injectI18n, FormattedMessage } from '@kbn/i18n/react';
+import { ES_TYPES } from '../../../common/es_types';
+import { PANEL_TYPES } from '../../../common/panel_types';
+
+const isFieldTypeEnabled = (fieldRestrictions, fieldType) =>
```

Comment:
Seems to be duplicated in `src/legacy/core_plugins/metrics/public/components/aggs/field_select.js`